### ISSUE TITLE
Add basic domain mode test coverage

### ISSUE THREAD

**Issue by tdiesler** [wildfly-extras/wildfly-camel#958]
It should be possible to test stuff in domain mode without the docker overhead

**Comment by tdiesler** [comment]
Blocked by https://issues.jboss.org/browse/WFARQ-1

**Comment by tdiesler** [comment]
Won't fix - see #1045
